gridstatus/nyiso.py

```diff
@@ -836,7 +838,7 @@ def _download_nyiso_archive(
 
             df = pd.read_csv(csv_url)
             df = _handle_time(df, dataset_name)
-            df["File Date"] = date.normalize()
+            df["File Date"] = self._get_load_forecast_file_date(date, verbose)
```

Comment:
this is inside of `def _download_nyiso_archive`, which isn't specific to load forecast

Reply:
Good point. I added a conditional so the `File Date` is only added when the caller requests it.